core/src/docs/concepts.rs

```diff
@@ -17,20 +17,26 @@
 
 //! The core concepts of OpenDAL's public API.
 //!
-//! OpenDAL provides a unified abstraction for all storage services.
+//! OpenDAL provides a unified abstraction that helps developers access all storage services.
 //!
 //! There are two core concepts in OpenDAL:
 //!
-//! - [`Builder`]: Build an instance of underlying services.
-//! - [`Operator`]: A bridge between underlying implementation detail and unified abstraction.
+//! - [`Builder`]: Builder accepts a series of parameters to set up an instance of underlying services.
+//! You can adjust the behaviour of underlying services with these parameters.
+//! - [`Operator`]: Developer can access underlying storage services with manipulating one Operator.
+//! The Operator is a delegate for underlying implementation detail, and provides one unified access interface,
+//! including `read`, `write`, `list` and so on.
+//! Through Operators, OpenDAL standardizes the access methods of different storage services into access to a file system-like interface.
```

Comment:
Our goal is not to provide a file system-like interface, and we should avoid giving our users this impression.

Reply:
sorry for the misleading. so what is the actual goal?

actually, the API design of Operator is similar to file system-like API during my reading of the code and the documentation.

Reply:
```suggestion
```

Reply:
sorry no offense for this. just feel confused when read doc and code.

Thank you again for your review